platform/ios/scripts/publish.sh

```diff
@@ -46,17 +46,7 @@ if [ -n "${CI:-}" ]; then
 fi
 
 step "Uploading ${ZIP} to s3…"
-REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
-aws s3 cp ../${ZIP} s3://mapbox/$REPO_NAME/ios/builds/ --acl public-read ${PROGRESS}
-echo "URL: https://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}"
+# Since this build is primarily for .CN customers, it will be hosted on .cn. 
 
-#
-# upload & update nightly
-#
-if [[ ${PUBLISH_VERSION} =~ "nightly" ]]; then
-    step "Updating ${PUBLISH_VERSION} to ${PUBLISH_STYLE}…"
-    GENERIC_NIGHTLY_FILENAME="mapbox-ios-sdk-${PUBLISH_VERSION}.zip"
-    aws s3 cp \
-        s3://mapbox/$REPO_NAME/ios/builds/${ZIP} \
-        s3://mapbox/$REPO_NAME/ios/builds/${GENERIC_NIGHTLY_FILENAME} --acl public-read ${PROGRESS}
-fi
+aws s3 cp ../${ZIP} s3://binary.mapbox.cn/mapbox-china-plugin/ios/builds/maps-sdk/ --acl public-read ${PROGRESS}
```

Comment:
The build will lives on `binary.mapbox.cn/mapbox-china-plugin`, it's a little bit confused, can we have a new directory for those builds?